docs_src/microservices/general/database/Ch-Postgres.md

```diff
@@ -25,6 +25,38 @@ The following EdgeX services depend on PostgreSQL for data storage and operate u
 - Security Proxy Auth
 - App Services (specifically for the Store and Forward feature)
 
+## Numeric Data Type Support
+
+To reduce disk consumption, numeric values are stored as `numeric` data type instead of `text` data type. For example, storing the number `128` as a string takes `3` bytes, whereas storing it as a numeric type takes only `2` bytes. See [Character Types][1] and [Numeric Types][2] for more details.
+
+The following data types are supported for storage as numeric values:
+
+- Int8, Int16, Int32, Int64, 
+- Uint8, Uint16, Uint32, Uint64
+- Float32, Float64
+
```

Comment:
Shall we also list array types here?

Reply:
No, this doc is only for storing the numeric value into the DB, the array type is a single reading value and the Postgres does not support to store it as numeric.